Mathlib/Computability/AkraBazzi/GrowsPolynomially.lean

```diff
@@ -17,6 +17,12 @@ statement of the Akra-Bazzi theorem: for the Akra-Bazzi theorem to hold, the fun
 satisfy the condition that `c₁ g(n) ≤ g(u) ≤ c₂ g(n)`, for u between b*n and n for any constant
 `b ∈ (0,1)`.
 
+## Main definitions and results
```

Comment:
This was moved from SumTransform.lean.

Reading the rest of the module docstring here, I realize adding this might just be repeating ourselves. I moved this point, rather than deleted it, because I wanted a PR that was as "simple" as possible in what it does.

If reviewers feel this can be deleted, I don't mind.

Reply:
I think it would make sense to just delete it.